projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.spec.ts

```diff
@@ -133,8 +133,82 @@ describe('SearchResultsRowComponent', () => {
 
   it('should pass node to badge component', () => {
     component.context = { row: { node: nodeEntry } };
+    fixture.detectChanges();
+
     const badgeElement = fixture.debugElement.query(By.css('aca-datatable-cell-badges'));
     expect(badgeElement).not.toBe(null);
-    expect(badgeElement.componentInstance.node).toBe(component.context.node);
+    expect(badgeElement.componentInstance.node).toBe(component.context.row.node);
+  });
+
+  it('should escape plain < and > in values', (done) => {
+    const customEntry: ResultSetRowEntry = {
+      entry: {
+        ...nodeEntry.entry,
+        name: '2 < 5 > 3',
+        search: { score: 5 }
+      }
+    } as ResultSetRowEntry;
+
+    component.context = { row: { node: customEntry } };
+    component.name$
+      .asObservable()
+      .pipe(first())
+      .subscribe(() => {
+        fixture.detectChanges();
+
+        const nameElement: HTMLSpanElement = fixture.debugElement.query(By.css('.aca-link.aca-crop-text')).nativeElement;
```

Comment:
In new code we use UnitTestingUtils so please try to use this instead of directly By.css. Also please create reusable function because '.aca-link.aca-crop-text' selector is repeated